test-tap/fixture/meta.js

```diff
@@ -1,9 +1,14 @@
 const {default: test, meta} = require('../..');
 
+test('meta is test.meta', t => {
+	t.is(meta, test.meta);
+});
+
 test('meta.file', t => {
 	t.is(meta.file, __filename);
 });
 
-test('test.meta.file', t => {
-	t.is(test.meta.file, __filename);
+test('meta.snapshotDirectory', t => {
+	t.regex(meta.snapshotDirectory, /.*snapshot-fixture.*/);
```

Comment:
Regex specific enough?

Reply:
Yea it's fine. Though you don't need the `.*`.